app/controllers/concerns/graphql_helper.rb

```diff
@@ -74,4 +127,31 @@ def match_partners_query(term)
 
     match_partners_response[:data][:match_partners]
   end
+
+  def my_collection_create_artwork_mutation_params(submission, current_user)
+    {
+      user_id: current_user,
```

Comment:
**suggestion:** We should use the `submission.user` ID instead of `current_user` in case we enable this feature for submissions created in Convection (or Volt) in the future.

Reply:
good catch, fixed